### DISCUSSION TITLE
Attendee feedback for events

### DISCUSSION THREAD

**Discussion by roysinn** [liedekef/events-made-easy#453] [Ideas]
I'm looking for a way to present a survey/feedback/evaluation form to attendees after an event. I envision a link in an EME email to attendees to an online form to record feedback for each attendee. Hope I didn't miss something in the documentation but there doesn't seem to be anything in EME. Just looking for ideas - some clever use (or misuse) of subscriptions, tasks, payments, etc. Any and all ideas muchly appreciated.

**Comment by liedekef** [comment]
EME is not a survey tool, so it is not designed for this. You could create a new event/rsvp form with custom fields, set that event as invisible, mail them the link and ask them to fill it out but that's it then. It may be sufficient of course.

**Reply by roysinn** [reply]
Thanks for quick reply, Franky. That is most helpful. I'm often surprised by all the things the EME **can** do, so I wanted to be sure that I had not missed anything. 

Continued thanks for EME and all your support. 

Roy
